src/web/app/pages-admin/admin-home-page/admin-home-page.component.html

```diff
@@ -79,10 +79,12 @@
           </tr>
         </tbody>
       </table>
+      <div class="top-padded">
       <button class="btn btn-primary" (click)="addAllInstructors()" id="add-all-instructors" [disabled]="activeRequests > 0 || isAddingInstructors">
```

Comment:
It will be great to add indentation here.

Reply:
Ok I will add identation and also update snapshots.